source/connect.txt

```diff
@@ -58,42 +58,51 @@ Considerations
 
 .. _vsce-connect-task:
 
-Create a Connection to a Deployment
------------------------------------
+Create a Connection
```

Comment:
Just curious, are there plans to separate these sections into individual task pages at some point?

Reply:
Yes definitely, I put a note on the PR and already created an epic for the effort, I hope to revisit this when some of the Q1 projects are knocked out: 

🔴 I made an epic[ DOCSP-36125](https://jira.mongodb.org/browse/DOCSP-36215) to capture the refactor needs for this page / repo:
Ideally, each of these tasks would be separate. Currently, we have several unrelated tasks such as edit / rename / copy on the connection page. These should be single tasks page under a "Manage your Connections" drawer.

Reply:
Oh sorry, missed that note in the PR. Thanks!